extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii

```diff
@@ -17,4 +17,29 @@ tags: {  # <amp-story-panning-media>
   amp_layout: {
     supported_layouts: FILL
   }
+  attrs: {
+    name: "x"
+    value_regex: "-?\\d+\\.?\\d*(%)"
+    trigger: {
+      also_requires_attr: "duration"
+    }
+  }
+  attrs: {
+    name: "y"
+    value_regex: "-?\\d+\\.?\\d*(%)"
+    trigger: {
+      also_requires_attr: "duration"
```

Comment:
How does this work? I don't see the `duration` in your test cases nor in this proto, does it need be removed?

Reply:
It does yes. Thank you for catching this.